ROADMAP.md

```diff
@@ -30,35 +30,26 @@ Status updates
 Short-term goals
 ----------------
 
-These are goals achievable within 3-6 months, tentatively done by Q3 2022.
+These are goals achievable within 3-6 months, tentatively done by Q2 2023.
 
-- **Stabilize the existing functionality and fix any major, high-impact, and blocking bugs.**<br>
-  Tool stability is something we take very seriously and users shouldn't run into any major showstopping issues.
+- **Make xk6-browser a part of standard distribution of k6.**<br>
 
   *How will we achieve this?*<br>
-  By manually testing the application in different scenarios, adding more automated tests, and addressing reports from internal and external users.
-  Any issues found that have a considerable user impact will be given highest priority and fixed ASAP.
+  xk6-browser will become an [experimental module](https://k6.io/docs/javascript-api/k6-experimental/) of k6, and thus available to use in load tests that utilize standard k6 release builds.
 
   *Definition of Done*<br>
-  This is difficult to gauge, but at some point we should stop running into any major issues on a frequent basis and should consider this goal as reached.
+  A load test script that imports `k6/x/browser` can be executed using the latest release build of k6. The browser-level APIs provided by xk6-browser are available for use within this script.
 
-
-- **Limited Alpha deployment and testing of the extension in k6 Cloud.**<br>
+- **Public beta in k6 Cloud.**
 
   *How will we achieve this?*<br>
-  Soon we will create a small-scale deployment for internal use and specific customers, meant to test the basic integration of the extension in k6 Cloud.
+  Cloud builds will start using k6 releases that include xk6-browser as an experimental module.
 
   *Definition of Done*<br>
-  When the extension is usable in k6 Cloud; i.e. scripts can run and test results are shown. Frontend changes are not required at this stage.
-
-
-Mid-term goals
---------------
-
-These are goals achievable within 6-12 months, tentatively done by mid 2023.
+  Load test executed in Cloud can use xk6-browser API by importing `k6/x/browser`.
 
 - **Transition our API to be async/`Promise` based.**<br>
```

Comment:
I think this could be broken down into two phases;
1. Convert the APIs that only work asynchronously (e.g. `WaitForNavigation` which is now async).
2. In a `n` months, convert the rest of the APIs to be async.

Reply:
> Convert the APIs that only work asynchronously

Not sure I follow, what would be included into this step?

Reply:
So the first phase, we would be prioritising and converting the APIs that most definitely need to be `async`. We need to sit down and do that prioritisation still. For example, i noticed that `Locator.click` is not async, but `Page.click` is `async`. I don't think we will convert many more APIs to `async` in the short term.

For the second phase (tbh) I'd only convert the rest of the APIs to `async` once we have `async`/`await` keywords.

Reply:
We might want to check Mihail's new PR for adding `async`/`await`: [k6@2783](https://github.com/grafana/k6/pull/2783). Then I believe we can reprioritize the async APIs higher if the experimental PR turns out to work nicely.